packages/build/package.json

```diff
@@ -60,7 +60,7 @@
     "@netlify/plugin-edge-handlers": "^1.8.0",
     "@netlify/plugins-list": "^2.0.0",
     "@netlify/run-utils": "^1.0.5",
-    "@netlify/zip-it-and-ship-it": "^2.0.0",
+    "@netlify/zip-it-and-ship-it": "^2.1.3",
     "@sindresorhus/slugify": "^1.1.0",
```

Comment:
Should https://github.com/netlify/build/blob/fbdb52d703e77889cdbf6a431758c28c0e346e1e/packages/functions-utils/package.json#L47 be updated too?

Reply:
Done 👍